### ISSUE TITLE
Address rule violation in JS.

### ISSUE THREAD

**Issue by andrewculver** [bullet-train-co/bullet_train#45]
```
16:35:20 js.1        | $ node esbuild.config.js --watch
16:35:21 js.1        | resolveDir /Users/andrewculver/Sites/asdf/app/javascript/controllers
16:35:21 js.1        | path ./**/*_controller.js
16:35:21 js.1        | ▲ [WARNING] "@charset" must be the first rule in the file
16:35:21 js.1        | 
16:35:21 js.1        |     node_modules/trix/dist/trix.css:300:0:
16:35:21 js.1        |       300 │ @charset "UTF-8";
16:35:21 js.1        |           ╵ ~~~~~~~~
16:35:21 js.1        | 
16:35:21 js.1        |   This rule cannot come before a "@charset" rule
16:35:21 js.1        | 
16:35:21 js.1        |     node_modules/trix/dist/trix.css:6:0:
16:35:21 js.1        |       6 │ trix-editor {
16:35:21 js.1        |         ╵ ^
16:35:21 js.1        | 
```

**Comment by pascallaliberte** [comment]
This issue might be fixed soon with the 2.0 release of trix (currently in beta) as a fix was recently [merged to basecamp/trix#main](https://github.com/basecamp/trix/pull/897).

Not sure it's worth fussing over with esbuild and postfix to re-write the `@charset` to appear at the top of the css file.

**Comment by pascallaliberte** [comment]
Maybe one direction would be to fix the trix.css inside `@bullet-train/fields` as part of the build process.
